terraspace/app/modules/eks-auth-sync/example/outputs.tf

```diff
@@ -0,0 +1,21 @@
+output "eks_token" {
+  value = data.aws_eks_cluster_auth.eks.token
+  sensitive = true
+}
+output "eks_host" {
+  value = data.aws_eks_cluster.eks.endpoint
+  sensitive = true
+}
+
+output "eks_cluster_ca_certificate" {
+  value = base64decode(data.aws_eks_cluster.eks.certificate_authority[0].data)
+  sensitive = true
+}
+
+output "fargate_iam_role" {
+  value = module.eks.fargate_profiles["default"].iam_role_name
+}
+
+output "managed_node_group_iam_role" {
+  value = module.eks.eks_managed_node_groups["test-ipfs-peer-subsys"].iam_role_name
```

Comment:
eventually, will we want to rename this nodegroup from `test-ipfs...` to something else?

Reply:
We can definitely use a better name than that.. I would just have some extra work migrating the states.
Do you have a naming suggestion?

Reply:
`<env>-ep-peer` or `ep-peer-<env>` - whatever our convention is currently ?

Reply:
Done!